client/src/components/Ticketing/ticketingmanager/Season/components/utils/ViewSeasonInfo.tsx

```diff
@@ -34,26 +50,91 @@ const ViewSeasonInfo = (props: ViewSeasonInfoProps) => {
 
   return (
     <header className='rounded-xl bg-white p-7 text-lg shadow-xl'>
-      <section className='flex flex-col text-center mb-5 justify-between tab:flex-row tab:flex-wrap'>
-        <h1 className='text-4xl mb-3 font-semibold'>Season Information</h1>
-        <button
-          className='bg-blue-500 hover:bg-blue-700 disabled:bg-gray-500 text-white font-bold py-2 px-10 rounded-xl'
-          onClick={() => setIsFormEditing(true)}
-        >
-          Edit
-        </button>
+      <section className='flex flex-col gap-3 text-center mb-5 justify-between tab:flex-row tab:flex-wrap'>
+        <h1 className='text-4xl font-semibold'>Season Information</h1>
+        <div className='flex flex-col gap-2 tab:flex-row tab:flex-wrap'>
+          <span
+            className={`${
+              activeSeasonSwitch ? 'bg-green-100' : 'bg-red-100'
+            } py-2 px-8 rounded-lg font-medium`}
+          >
+            {activeSeasonSwitch ? 'ACTIVE' : 'INACTIVE'}
+          </span>
+          <Tooltip
+            title={<p style={{fontSize: '1.2rem'}}>Edit</p>}
+            placement='top'
+            arrow
+          >
+            <button
+              className='flex justify-center items-center bg-gray-400 hover:bg-gray-500 disabled:bg-gray-500 text-white font-bold px-3 py-2 rounded-xl'
```

Comment:
I'd replace the `px-3 py-2` with just a `w-11`. That way its a bit more square, and I think the other flex stuff will auto do the height.

Reply:
Will do!